src/irmin/store.ml

```diff
@@ -434,7 +434,7 @@ module Make (P : S.PRIVATE) = struct
             if Tree.equal vx vy then skip_key key
             else (
               changed_key key;
-              fn @@ `Updated ((x, vx), (y, vy)) ) )
+              fn @@ `Updated ((x, vx), (y, vy))))
```

Comment:
Here's a case where I'll miss this feature :cry:

Reply:
your editor can highlight the enclosing parenthesis :p (I don't think that there is such a whitespace convention in heavy-parenthesis languages such as lisp)

Reply:
In my defense, parentheses in lisp are a single syntactic construct, whereas this feature mostly distinguishes units/tuples from grouping parens. :wink: